lib/collection/src/shards/shard_holder.rs

```diff
@@ -323,6 +361,34 @@ impl ShardHolder {
         Ok(())
     }
 
+    pub fn resharding_filter(&self) -> Option<Filter> {
```

Comment:
How about something like this, to clarify the semantics a bit?

```suggestion
    pub fn resharding_filter_exclude_migrated(&self) -> Option<Filter> {
```

Reply:
Dunno. Does not make much difference, IMO. Probably better to add documentation to the method.

Reply:
```suggestion
    /// Create a filter that excludes points migrated to a different shard, as part of resharding.
    pub fn resharding_filter(&self) -> Option<Filter> {
```